libgadget/fof.c

```diff
@@ -401,13 +378,25 @@ void fof_label_primary(ForceTree * tree, MPI_Comm Comm)
         treewalk_run(tw, NULL, PartManager->NumPart);
 
         t1 = second();
-
+        /* Just in case we missed some during HEAD building.*/
```

Comment:
If it is abnormal, what about adding an error or a warning?
It it is normal, then briefly describe how it happens?

Reply:
It was "just in case" when I introduced it but in the final commit it comes from a fairly normal race. I will add a comment.